lib/matplotlib/animation.py

```diff
@@ -234,12 +234,14 @@ def grab_frame(self, **savefig_kwargs):
             # frame format and dpi.
             self.fig.savefig(self._frame_sink(), format=self.frame_format,
                              dpi=self.dpi, **savefig_kwargs)
-        except RuntimeError:
+        except (RuntimeError, IOError):
             out, err = self._proc.communicate()
             verbose.report('MovieWriter -- Error '
                            'running proc:\n%s\n%s' % (out,
                                                       err), level='helpful')
-            raise
+            raise IOError('Error saving animation to file. '
+                          'Stdout: {0} StdError: {1}. It may help to re-run '
+                          'with --verbose-debug.'.format(out, err))
```

Comment:
The original exception is now getting swallowed. Would probably be nice to at least have it logged through `verbose.report()`?